plugins/scaffolder/src/next/Router/Router.tsx

```diff
@@ -95,7 +106,10 @@ export const Router = (props: PropsWithChildren<NextRouterProps>) => {
           <SecretsContextProvider>
             <TemplateWizardPage
               customFieldExtensions={fieldExtensions}
-              FormProps={props.FormProps}
+              FormProps={{
+                ...props.FormProps,
+                layouts: customLayouts,
```

Comment:
I think that these should just be on the `TemplateWizardPage` as they're not props that are on the `Form` especially, they're props that we convert into some concept known on the `Form` itself. They're pretty similar the `customFieldExtensions`.

Reply:
@benjdlambert agreed